src/internal/message/spinner.go

```diff
@@ -60,10 +60,8 @@ func (p *Spinner) Debugf(format string, a ...any) {
 }
 
 func (p *Spinner) Stop() {
-	if p.spinner != nil && p.spinner.IsActive {
```

Comment:
Yeah this was really a debug thing for me when I was doing that because we had some random spinner duplications.  It was a real problem and we still should have some type of singleton logic here eventually because you actually can't have more than one spinner at once, but it feels like that should exist in the upstream vs here. Also, needed this to not be there so we could properly call a `Stop()` when `Success()` or `Fatal()` weren't applicable.